src/ostorlab/agent/agent.py

```diff
@@ -40,6 +44,16 @@ class MaximumCyclicProcessReachedError(exceptions.OstorlabError):
     """The cyclic process limit is enforced and reach set value."""
 
 
+def _setup_remote_logging():
+    gcp_logging_credential = os.environ.get(GCP_LOGGING_CREDENTIAL_ENV)
+    if gcp_logging_credential is not None:
+        info = json.loads(gcp_logging_credential)
+        credentials = service_account.Credentials.from_service_account_info(info)
+        # Instantiates a client
```

Comment:
Can we please handle the case were `json.loads` fails?
Also the comment: `# Instantiates a client` is not really useful.